### DISCUSSION TITLE
I built a Destiny 2 aim bot using ecal and yolov5

### DISCUSSION THREAD

**Discussion by rlewkowicz** [eclipse-ecal/ecal#928] [Show and tell]
https://hireryan.today/2023/01/02/just-a-better-gaming-chair-honestly-destiny-2-aim-bot-using-yolov5/

The code is hot trash, this is just a personal project. It needs a ground up rewrite. But this is an architectural overview of it.

**Comment by rex-schilasky** [comment]
Interesting use case !

**Comment by FlorianReimold** [comment]
Thank you very much for sharing! I love seeing people use eCAL for scenarios that are entirely unrelated to automated driving!
